Documentation/contributor-guide/bump_etcd_version_k8s.md

```diff
@@ -66,7 +66,7 @@ You can refer to the guide [here](https://github.com/kubernetes/community/blob/m
     match: BUNDLED_ETCD_VERSIONS\?|
     ```
 
-2. In `cluster/images/etcd/Makefile`, include the new version in `BUNDLED_ETCD_VERSIONS` and update the `LATEST_ETCD_VERSION` as well (the image tag will be generated from the `LATEST_ETCD_VERSION`). Update `GOLANG_VERSION` if necessary.
+2. In `cluster/images/etcd/Makefile`, include the new version in `BUNDLED_ETCD_VERSIONS` and update the `LATEST_ETCD_VERSION` as well (the image tag will be generated from the `LATEST_ETCD_VERSION`). Update `GOLANG_VERSION` according to the version used to compile that release version (can be seen in the CHANGELOG).
```

Comment:
We need to update the go version in `build/dependencies.yaml` if needed, and ensure the go version matches the go version used in `cluster/images/etcd/Makefile`